app/templates/main/new_org.html

```diff
@@ -0,0 +1,18 @@
+{% extends "main/base.html" %}
+{% block title %}Ajouter une organisation{% endblock %}
```

Comment:
I think it should read "Ajouter une association" (same below in the `<h1>` title)